datalake/src/test/java/com/sequenceiq/datalake/controller/sdx/SdxControllerTest.java

```diff
@@ -190,6 +197,21 @@ void refreshDatahubs() {
         verify(sdxService, times(1)).refreshDataHub(SDX_CLUSTER_NAME, null);
     }
 
+    @Test
+    void validateBackupStorage() {
+        SdxCluster sdxCluster = getValidSdxCluster();
+        sdxCluster.setClusterName(SDX_CLUSTER_NAME);
+        SdxBackupLocationValidationRequest sdxBackupLocationValidationRequest = new SdxBackupLocationValidationRequest(SDX_CLUSTER_NAME);
+        when(storageValidationService.validateBackupStorage(any())).thenReturn(null);
```

Comment:
Please
* Use proper argument matching for `validateBackupStorage()` to match `sdxCluster`. (As I see, you have a `verify()` for this, but it is better to include the appropriate matching here when defining the stub behavior.)
* Return a proper `ValidationResult`. (An empty object is sufficient, the actual validation status is not crucial here.)

Reply:
addressed.